SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/Spatial2WorkerSmallInterestMap.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/Spatial2WorkerSmallInterestMap.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "GameFramework/PlayerStart.h"
+#include "SpatialGDKFunctionalTests/Public/SpatialCleanupConnectionTest.h" //didn't spot this during review, but I would argue this one is "SpatialGDK" as well
```

Comment:
Is the trailing comment here necessary?

Reply:
theoretically not, I have left a few comments on the test themselves and not sure what to do with them.
I guess I could create a cleanup ticket where I mention these things...

Reply:
https://improbableio.atlassian.net/browse/UNR-4981 created these
Removed this comment, left the one in authority map, as I think that one is less meta